### PR TITLE
Adds React with Create React App

### PR DESCRIPTION
**Pull request by hottmanmichael**
Used create react app to build a directory called `react-challenge` that will be hooked up to the same `server` backend - None of the challenge has been written in react yet, that will be a separate PR

### CODE REVIEW

File: react-challenge/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
```
Comment (bryan-pana) [inline react-challenge/.gitignore:19]: Should there be a plain old `.env` here?

**Review by bryan-pana — APPROVED** [review]

File: react-challenge/.gitignore (hunk 1 above)
Comment (severin2) [reply react-challenge/.gitignore:19]: I think the standard usage is that we check a .env with non-secret standard values into the repo that is overridden by specific environment files that are never checked in

**Review by severin2 — CHANGES_REQUESTED** [review]
i think this would end up being the first project that uses yarn instead of npm, do we intend to use npm when this challenge gets built out?

**Review by dglunz — APPROVED** [review]

File: react-challenge/.gitignore (hunk 1 above)
Comment (hottmanmichael) [reply react-challenge/.gitignore:19]: I don't think it matters cause there's not actually any .env setup needed for the code challenge

**Comment by hottmanmichael** [comment]
> i think this would end up being the first project that uses yarn instead of npm, do we intend to use npm when this challenge gets built out?

@severin2 `yarn` is just the default for create-react-app - But both can be used interchangeably, so either works to build the project

**Pull request merged**